.github/workflows/build-test-measure.yml

```diff
@@ -199,8 +199,6 @@ jobs:
   e2e-test-js:
     name: 'E2E test: JS'
     runs-on: ubuntu-latest
-    # Tests are a bit flaky at the moment.
-    continue-on-error: true
```

Comment:
Also now making the E2E job a required check in hopes we no longer see frequent flaky tests in the near future.